demos/form-control/input2.php

```diff
@@ -14,24 +14,24 @@
 /** @var \Atk4\Ui\App $app */
 require_once __DIR__ . '/../init-app.php';
 
-\Atk4\Ui\Header::addTo($app, ['Disabled and read only form controls (normal / readonly / disabled)']);
+\Atk4\Ui\Header::addTo($app, ['Disabled and read only form controls (normal / readOnly / disabled)']);
```

Comment:
Where does `normal` come from?

To the SUI/FUI case - we should use the canonical form from SUI/FUI docs.

Reply:
@mvorisek This is just a description for the demo to show normal controls, or readonly or disabled controls. No code implication.

Reply:
then the original should stay